### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by Harry-Colo-Codes**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Welcome to Potionfy</title>
 </head>
 <body>
-    
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (Hermione-Colo-Codes) [inline index.html]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Comment (Harry-Colo-Codes) [reply index.html]: Sure, can do!

**Review by Hermione-Colo-Codes — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by Hermione-Colo-Codes — APPROVED** [review]
Well done Harry!

**Pull request merged**